src/controllers/userController.ts

```diff
@@ -0,0 +1,24 @@
+import { RequestWithUser } from '../types/requests.js';
+import { NextFunction, Response } from 'express';
+import { UserService } from '../services/userService.js';
+import { BaseUser } from '../types/user.js';
+import { BaseController } from './baseController.js';
+
+
+export class UserController extends BaseController {
+    private userService: UserService;
+    constructor(userServiceInjected: UserService) {
+        super();
+        this.userService = userServiceInjected;
+    }
+    async createHandler(req: RequestWithUser, res: Response, next: NextFunction) {
+        const user: BaseUser = req.body;
+        try {
+            const createdUser = await this.userService.create(user);
```

Comment:
Let's think on this.

You execute the statement to insert the user to database. Most likely, in returned data we will have the sensitive information. So, question is, should do like this?

Another question, what can we do here to avoid Information leakage?

So, the straight forward way is just to create mapper for this.
I.e. create class like `UserMapper` in the `/api/mappers` and method 
```
static toClient(entity: IUserEntity): IUserClient {
     return {
        uuid: entity.uuid,
        ...
     }
}
```

and in the controller you can do, something like

```
...
const createdUser = await this.userService.create(user);
...

this.success(res, UserMapper.toClient(createdUser));
...
```